helm/superset/values.yaml

```diff
@@ -800,7 +800,7 @@ redis:
     ## docker registry secret names (list)
     # pullSecrets: nil
     ##
-    ## Configure persistance
+    ## Configure persistence
```

Comment:
@hugo-syn because you've edited the Helm chart configs you'll also have to bump the chart version (as indicated by the failing CI check):

```
 ✖︎ superset => (version: "0.11.2", path: "helm/superset") > chart version not ok. Needs a version bump! 
```

Reply:
I just bumped the version